GCTC-NTGC/gc-digital-talent
# Update Talent Cloud report link

Resolves #3517.

### Steps to test

1. Run `npm run production` in frontend/talentsearch/
2. Navigate to '/logged-out'
3. Click on _Talent Cloud report_ link
4. Verify link goes to Talent Cloud report page and not 404
